vispy/color/colormap.py

```diff
@@ -1118,26 +1114,24 @@ def get_colormap(name, *args, **kwargs):
         ``RdYeBuCy`` (``RedYellowBlueCyan``) colormap you must import and
         instantiate it directly from the ``vispy.color.colormap`` module.
 
+    .. versionchanged: 0.13.1
+
+        Remove deprecation for ``cubehelix`` (``CubeHelixColormap``),
+        ``single_hue`` (``SingleHue``), ``hsl`` (``HSL``), ``husl`` (``HSLuv``),
+        ``diverging`` (``Diverging``), and ``RdYeBuCy`` (``RedYellowBlueCyan``)
+        colormaps, which are supported.
+
+        No longer accept args/kwargs at all.  Now passing additional arguments
+        will raise an exception.
+
```

Comment:
```suggestion
```

In my opinion there is no need to note the version where a deprecation was removed...but maybe I'm the only one (CC @brisvag). At a minimum I think the long list of colormaps could be excluded from this message and change it to "Additional arguments and keyword arguments are no longer accepted. Only colormap instances (not classes) are returned."

And if the version change note is kept the version should be changed to 0.14.0.

Reply:
I agree, this seems overly cautious and we could just leave the comment out :)